docs/check-redirects/check-redirects.test.js

```diff
@@ -1,4 +1,5 @@
 import { Volume, createFsFromVolume } from 'memfs';
+
```

Comment:
I was wondering why the vercel bot keeps spamming under this PR. It turns out it's because of this change.